tests/test_inherits_path_option.py

```diff
@@ -26,26 +31,23 @@ def write_and_run_simple_pex(inheriting=False):
         pb.set_executable(os.path.join(td, "exe.py"))
         pb.freeze()
         pb.build(pex_path)
-        yield run_simple_pex(pex_path, env={"PEX_VERBOSE": "1"})[0]
+        stdout, _ = run_simple_pex(pex_path, env={"PEX_VERBOSE": "1"})
+        yield stdout.decode("utf-8")
```

Comment:
We didn't call `.decode()` before. Instead, we used `str()` on the object at each callsite. This is more explicit